plugins/module_utils/template.py

```diff
@@ -269,3 +269,13 @@ def get_node_settings_object(self, uuid=None, name=None, fail_module=False):
         if uuid or name:  # Query a specific object
             return self.get_object_by_key_value_pairs("Node Settings", existing_objects, [KVPair("uuid", uuid) if uuid else KVPair("name", name)], fail_module)
         return existing_objects  # Query all objects
+    def get_route_map_policy_for_multicast_uuid(self, route_map_policy_for_multicast_name):
```

Comment:
CI is failing because of the black formatting check.

Reply:
resolved